go-controller/go.mod

```diff
@@ -20,11 +19,15 @@ require (
 	github.com/juju/errors v0.0.0-20200330140219-3fe23663418f // indirect
 	github.com/juju/testing v0.0.0-20200706033705-4c23f9c453cd // indirect
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v0.0.0-20200626054723-37f83d1996bc
+	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
 	github.com/miekg/dns v1.1.31
 	github.com/onsi/ginkgo v1.15.0
 	github.com/onsi/gomega v1.10.1
+	github.com/ovn-org/libovsdb v0.3.0
```

Comment:
I see you have 0.3.1 here https://github.com/ovn-org/libovsdb/releases/tag/v0.3.1

Reply:
yep, there wasn't any sense in bumping now because you'll need v0.4.0, which isn't ready yet